turbine/src/cluster_nodes.rs

```diff
@@ -489,7 +546,7 @@ fn get_retransmit_peers<T>(
 
 // Returns the parent node in the turbine broadcast tree.
 // Returns None if the node is the root of the tree.
-fn get_retransmit_parent<T: Copy>(
```

Comment:
I support this name change, but highly encourage splitting cosmetic vs. functional changes in the future

Reply:
Fair point, I will revert the name change here.